zulipterminal/cli/run.py

```diff
@@ -244,6 +247,11 @@ def main(options: Optional[List[str]]=None) -> None:
         print('Zulip Terminal ' + ZT_VERSION)
         sys.exit(0)
 
+    if args.list_themes:
+        print('\nThemes available:-',)
+        print('\t' + "\n\t".join(all_themes()) + '\n')
```

Comment:
We haven't used tabs in the rest of this file; we could do with standardizing/simplifying, but for now could you use spaces for consistency?

I'm not sure we need leading/trailing blank lines.

You could also indicate how the theme is specified after listing the themes?

Reply:
By specifying, do you mean specifying how to change the theme to the other ones using `--theme` or `-t`?.

Reply:
Yes, via that and possibly also the config file option.